src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -4008,6 +4008,8 @@ void MusicXMLParserPass2::clef(const QString& partId, Measure* measure, const Fr
             clef = ClefType::C5;
         } else if (line == 4) {
             clef = ClefType::C4;
+        } else if (i == -1 && line == 4) {
+            clef = ClefType::C4_8VB;
```

Comment:
But now this `i == -1` case won't be reached

```suggestion
        } else if (line == 4) {
            if (i == -1) {
                clef = ClefType::C4_8VB;
            } else {
                clef = ClefType::C4;
            }
```

Reply:
Also might be good to update line 3960 (`if (i && !(c == "F" || c == "G")) { …`)

Reply:
Donr